src/BackgroundTimer.js

```diff
@@ -0,0 +1,39 @@
+import {useState, useEffect} from 'react';
+
+function BackgroundTimer() {
+    // sets initial value for inactivity timer to 15 seconds
+    const [counter, setCounter] = useState(15)
+    // sets initial value for user activity to false
+    const [userActivity, setUserActivity] = useState(false);
+
+    useEffect(()=>{
+        setUserActivity(false)
+        if (userActivity==true){
+          //pauses current timeout when user activity is detected
+          clearTimeout()
+          //resets background timer to 15 seconds
+          setCounter(15)
+        } else if (counter > 0){
+          //counts down by 1 second
+          setTimeout(() => setCounter(counter - 1), 1000)
+          console.log(counter)
+        } else {
+          alert("Hi!")
```

Comment:
We'll also want to change this alert message from its current placeholder before merging I think

Reply:
Alert has been changed